package.json

```diff
@@ -94,10 +94,11 @@
     "dist"
   ],
   "dependencies": {
+    "@popperjs/core": "^2.9.2",
```

Comment:
For the `Dropdown` component, comparing the old and the new Storybooks it looks like the 'click outside to close' behaviour doesn't work anymore and the spacing is not the same between the 'triggerer' and the dropdown lists.